File: Solution/CSharp/283.MoveZeroes.cs
public class Solution {
    public void MoveZeroes(int[] nums) {
        int length = nums.Length;
        for (int i = 0; i < length; i++) {
            for (int j = i; j < length; j++) {
                if (nums[i] == 0) {
                    int temp = nums[i];
                    nums[i] = nums[j];
                    nums[j] = temp;
                }
            }
        }
    }
}

File: Solution/Javascript/283. Move Zeroes.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var moveZeroes = function (nums) {
    let n = nums.length;
    let left = 0;
    let right = 0;
    while (right < n) {
        if (nums[right]) {
            let temp = nums[left];
            nums[left] = nums[right]
            nums[right] = temp;
            left++;
        }
        right++;
    }
};